src/runner.c

```diff
@@ -36,6 +48,14 @@ int main(int argc, char **argv)
 		new_argv[wrapper->argc] = NULL;
 	}
 
+	char *env_prefix = "SUID_WRAPPER_";
+	char *new_env[] = {
+		ssprintf("%sUID=%d", env_prefix, getuid()),
+		ssprintf("%sGID=%d", env_prefix, getuid()),
+		ssprintf("%sNAME=%s", env_prefix, getpwuid(getuid())->pw_name),
```

Comment:
The linux equivalent env var is `$USER`. Would it be good idea to use the same so it becomes `SUID_WRAPPER_USER` instead?

Reply:
You are right. I changed it to SUID_WRAPPER_USER.